content/javascript/concepts/operators/operators.md

```diff
@@ -153,6 +149,14 @@ console.log(isPenguin);
 // Output: false
 ```
 
+## Unary Operator
+
+A unary operator, such as the not operator (`!`), is an operator that is applied to one term or variable. Arithmetic operators are also known as binary operators, because they operate on two terms. There are also unary operators in JavaScript that are names rather than symbols, such as the `typeof` operator.
```

Comment:
```suggestion
A unary operator, such as the not operator (`!`), is an operator that is applied to one term or variable. Arithmetic operators are also known as binary operators because they operate on two terms. There are also unary operators in JavaScript that are names rather than symbols, such as the `typeof` operator.
```